README.md

```diff
@@ -870,7 +877,131 @@ DB::unprepared(
 );
 ```
 
-Tip given by [@w3Nicolas](https://twitter.com/w3Nicolas/status/1447902369388249091)
+### The crossJoinSub method of the query constructor
+Using the CROSS JOIN subquery
+```php
+use Illuminate\Support\Facades\DB;
+
+$totalQuery = DB::table('orders')->selectRaw('SUM(price) as total');
+
+DB::table('orders')
+    ->select('*')
+    ->crossJoinSub($totalQuery, 'overall')
+    ->selectRaw('(price / overall.total) * 100 AS percent_of_total')
+    ->get();
+```
+
+Tip given by [@PascalBaljet](https://twitter.com/pascalbaljet)
+
+### Order by Pivot Fields
+`BelongsToMany::orderByPivot()` allows you to directly sort the results of a BelongsToMany relationship query.
+```php
+class Tag extends Model
+{
+    public $table = 'tags';
+}
+
+class Post extends Model
+{
+    public $table = 'posts';
+
+    public function tags()
+    {
+        return $this->belongsToMany(Tag::class, 'posts_tags', 'post_id', 'tag_id')
```

Comment:
naming convention for join tables is singular models in alphabetical order. So this

>          return $this->belongsToMany(Tag::class, 'posts_tags', 'post_id', 'tag_id')

should be using table `post_tag` and that would mean you wouldn't need to specify the table name or keys.

>      return $this->belongsToMany(Tag::class)

Reply:
@iammikek I would accept a pull request for this change